src/enums.ts

```diff
@@ -0,0 +1,161 @@
+import { z } from 'zod'
+
+export enum TransferType {
+  TransferIn = 'TransferIn',
+  TransferOut = 'TransferOut',
+}
+export const transferTypeSchema = z.nativeEnum(TransferType)
```

Comment:
Do we need to export a zod schema for enums? I don't think we use an enum schema separately anywhere and we can just inline z.nativeEnum(enumType) in the request / response body definitions.

Reply:
I think we should do it just so we can say 'All types also have a corresponding zod schema'. Its a little extra code and may not be used but it makes the API very clear